src/js/models/maps/viewfinder/ViewfinderModel.js

```diff
@@ -146,6 +154,21 @@ define(
         });
       },
 
+      /**
+       * Select a ZoomPresetModel from the list of presets and navigate there.
+       * @param {ZoomPresetModel} preset A user selected preset for which to 
+       * enable layers and navigate.
+       */
+      selectZoomPreset(preset) {
```

Comment:
should selecting a preset hide any visible layers first?

Reply:
The showing/hiding happens in order of the layers in this.allLayers, this is currently looping through all layers not just the ones in the zoom preset

Reply:
I was... not thinking🥲

Reply:
I'll add a comment, I think you're right that it's not exactly clear